onnxruntime/core/framework/debug_node_inputs_outputs_utils.h

```diff
@@ -75,6 +77,7 @@ struct NodeDumpOptions {
     TensorProtoFiles,
   } data_destination{DataDestination::StdOut};
 
+  std::string file_suffix = "";
```

Comment:
nit: `string s = ""` is not necessary as the string default ctor makes an empty string. if explicitly initializing it, can we spell it `string s{}` for consistency?